### PR TITLE
Fix cabal constraints

### PR DESCRIPTION
**Pull request by 4e6**
Fix build constraints

## classy-prelude-conduit >=1

```
/home/dbushev/projects/Aviora/stackage-curator/Stackage/Curator/UploadDocs.hs:113:18: Not in scope: ‘askRunBase’

/home/dbushev/projects/Aviora/stackage-curator/Stackage/Curator/UploadDocs.hs:114:20:
    Not in scope: ‘runConcurrently’

/home/dbushev/projects/Aviora/stackage-curator/Stackage/Curator/UploadDocs.hs:115:13:
    Not in scope: data constructor ‘Concurrently’

/home/dbushev/projects/Aviora/stackage-curator/Stackage/Curator/UploadDocs.hs:117:28:
    Not in scope: data constructor ‘Concurrently’

/home/dbushev/projects/Aviora/stackage-curator/Stackage/Curator/UploadDocs.hs:240:14:
    Not in scope: type constructor or class ‘MonadBaseUnlift’
```

## aeson >= 1 

```
/home/dbushev/projects/Aviora/stackage-curator/app/stackage-build-plan.hs:4:18:
    Could not find module ‘Data.Aeson.Text’
    Perhaps you meant
      Data.Aeson.TH (from aeson-0.11.3.0@aeson_FkJ0QVv64AF4wUlW9izgcW)
      Data.Aeson.Types (from aeson-0.11.3.0@aeson_FkJ0QVv64AF4wUlW9izgcW)
    Use -v to see a list of the files searched for.
```

### CODE REVIEW

File: Stackage/PackageIndex.hs [hunk 1]
```diff
@@ -110,10 +109,6 @@ data SimplifiedPackageDescription = SimplifiedPackageDescription
     }
     deriving Generic
 
-#if !MIN_VERSION_base(4, 9, 0)
```
Comment (4e6) [inline Stackage/PackageIndex.hs]: This should be the test of cabal version `!MIN_VESION_Cabal(2,0,0)`, but stackage-curator dropped Cabal <2 support

**Comment by snoyberg** [comment]
Thanks! I'm not opposed to PRs like this, but I'm surprised you're tripping on this. How are you trying to build and use this tool?

**Comment by 4e6** [comment]
Hi, I'm the guy behind [typeable/stackage2nix](https://github.com/typeable/stackage2nix). I was adding some tests building the project against different resolvers when encountered this (these particular issues were from `lts-6` branch). 
I'm not interested in supporting GHC7 in general because there are a bunch of weird issues in an interaction of old Cabal and Nix. Just cleaning up, since I already got my hands dirty :)

**Comment by 4e6** [comment]
Ah, also, see you on the Fby!

**Comment by snoyberg** [comment]
Cool, I look forward to seeing you! I'd be happy to hear more about stackage2nix if you want to chat about it in person.

**Pull request merged**